images/dockerfiles/stellar-core-telegraf/input-exec/get-app-state-synced.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
```

Comment:
the first section is totally over-engineered and requires too much bash. bash is terrible.

Reply:
i removed the switch. please comment on the exact line so github could mark them as "outdated" once it's fixed